resources/assets/less/bem/osu-page-header.less

```diff
@@ -42,6 +42,12 @@
     }
   }
 
+  &--two-col {
+    @media @desktop {
+      flex-wrap: nowrap !important;
```

Comment:
I think setting actual 60/40 width on `@desktop` for each boxes work as well instead of using !!!!!!!!!!!importantt!!!!